src/firefly/java/edu/caltech/ipac/firefly/core/background/JobInfo.java

```diff
@@ -204,6 +204,7 @@ public SrvParam getSrvParams() {
 //====================================================================
 
     public static record Error ( int code, String msg){}
+    public static record Result(String href, String mimeType, String size){};
```

Comment:
Is there any more result data that we could keep?

Reply:
The schema is hard to read.  But, after reading it again, I found an attribute I missed; `xlink:type`.  I could add this.  But, I have no idea how it's used.  It's not documented anywhere except in the schema.